src/database/DataTypes/Name.js

```diff
@@ -4,6 +4,8 @@
  */
 
 import Realm from 'realm';
+import { UIDatabase } from '../index';
+import { PREFERENCE_KEYS } from '../utilities/preferenceConstants';
```

Comment:
This isn't ideal- adding UIDatabase import is a circular reference .. 

Alternatives:

- Put it into the `DataTableRow` (which is calling the getter on an instance of this class). Not great as it means this getter is actually not correct
- Put it into the component which calls `DataTableRow` and pass as some sort of `option` prop... not great as there is already plenty of props and the condition for enabling/disabling a row is already really crazy in `DataTableRow`, but the advantage is that this menans there's one db call

Reply:
would it be crazy to lookup preferences when the app starts and store them globally?
Update the state on incoming sync?